kani-driver/src/unsound_experiments.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+#![cfg(feature = "unsound_experiments")]
+use std::ffi::OsString;
+use structopt::StructOpt;
+#[derive(Debug, StructOpt)]
+pub struct UnsoundExperimentArgs {
+    /// Zero initilize variables.
+    /// This is useful for experiments to see whether assigning constant values produces better
+    /// performance by allowing CBMC to do more constant propegation.
+    /// Unfortunatly, it is unsafe to use for production code, since it may unsoundly hide bugs.
+    /// Marked as `unsound` to prevent use outside of experimental contexts.
+    #[structopt(long, hidden_short_help(true), requires("enable-unstable"))]
+    pub unsound_experiment_zero_init_vars: bool,
+}
+
+impl UnsoundExperimentArgs {
+    pub fn process_args(&self) -> Vec<OsString> {
+        self.print_warnings();
```

Comment:
Will this get printed even without the option enabled?

Reply:
`print_warnings` only prints for enabled options